packages/nimble-components/src/wafer-map/modules/event-coordinator.ts

```diff
@@ -11,13 +11,12 @@ export interface EventCoordinatorCallbacks {
  * EventCoordinator deals with user interactions and events
  */
 export class EventCoordinator {
-    private readonly zoomHandler: ZoomHandler;
-    private readonly hoverHandler: HoverHandler;
-
-    public constructor(private readonly wafermap: WaferMap) {
-        this.zoomHandler = new ZoomHandler(wafermap);
-        this.hoverHandler = new HoverHandler(wafermap);
+    private readonly zoomHandler = new ZoomHandler(this.wafermap);
+    private readonly hoverHandler = new HoverHandler(this.wafermap);
+    public constructor(private readonly wafermap: WaferMap) {}
```

Comment:
So this is a funny one. Normal JS supports class members nowadays but the pattern you're doing here (where it is expected the constructor runs before the properties) does not actually happen in JS. See: https://jsbin.com/dekatuc/1/edit?js,console and notice that undefined is printed because the ZoomHandler is constructed before the constructor runs, not after.

The reason this works is because TypeScript actually generates code closer to [what you had originally](https://www.typescriptlang.org/play?#code/MYGwhgzhAEDyAuALApgJwCqIJYDsDm0A3gFDRnTEC+xokMmuBJ50wA9jhPKgK7DxtUACjZI0DfAC44YjNnwBKItWrIAHgAdB8VuCjQAogDdkOeAGE2ggCa4wA1EVLkNqLEfvJoqZGGscQAE9oAC8rAFsACTAcaxA0aABeaBxkAHdoCTwhJCwIADo0sAAzNHCwDQUAbmcyV3dPb19-HCDoRDYTVGjY+Mdk1IysnOwCotLUcsqalg0eACMQLGBWDi5efkEheo94Lx8-AODxsorpBBQ5RiVCFSA)

So in just this case I would recommend keeping the constructor for initializing these two class members (zoom and hover handler) so the source better aligns with what is generated.